test/geometry/test_line.py

```diff
@@ -3,33 +3,40 @@
 from torch.autograd import gradcheck
 
 from kornia.geometry.line import ParametrizedLine, fit_line
-from kornia.testing import assert_close
 
 
-class TestParametrizedLine:
+class BaseTester:
+    def assert_close(self, actual, expected, rtol=None, atol=None):
```

Comment:
@MrShevan this is another common pattern i found for testing

Reply:
May we merge this [PR](https://github.com/kornia/kornia/pull/1809) into master and merge new master into this branch? 
Then we can add this logic to method `assert_close` from existing `BaseTester` from `kornia.testing`?
```
if isinstance(actual, torch.nn.Parameter):
    actual = actual.data
if isinstance(expected, torch.nn.Parameter):
    expected = expected.data
```

Reply:
defenitely